docs/tcp_LAN_WAN_transport.rst

```diff
@@ -0,0 +1,43 @@
+TCP Transport
+==============
+
+Fast RTPS transport layer could be adapted to specific network requirements. UDP transport is optimal for localhost applications but in order to interact within a LAN or WAN a TCP transport may be required. Here we explain how Shapes Demo should be set up in order to fit some network specific layouts.
```

Comment:
UDP is optimal for localhost? Not only for that, also for local networks. TCP must be used when jumps between networks (with routers in the middle) are needed. Please, try to write this idea.

Reply:
I try to rephrase it as:
UDP transport is optimal for local network applications but in order to interact on a Wide Area Network (WAN), where one or several NAT mappings may be enforced, a TCP transport would be more suitable.